docs/UserGuide.md

```diff
@@ -232,8 +235,8 @@ Allows you to view details, such as basic information, medical condition and rem
 * The index **must be a positive integer** 1, 2, 3, …​
 
 **Examples**:
-* `list_p` followed by `view_p 2` views the 2nd patient in the patient list.
-* `find_p Jonathan` followed by `view_p 1` views the 1st patient in the results of the `find_p` command.
+* Entering `list_p` followed by `view_p 2` views the 2nd patient in the patient list.
```

Comment:
instead of 'views' maybe 'shows the details of' would be more grammatically correct.